src/app/addwidget/add-widget.module.ts

```diff
@@ -1,14 +0,0 @@
-import { NgModule } from '@angular/core';
-
-import { AddWidgetComponent } from './add-widget.component';
-
-@NgModule({
-  declarations: [AddWidgetComponent],
-  imports: [
-  ],
-  exports: [
-    AddWidgetComponent
-  ]
-})
-export class AddWidgetModule {
```

Comment:
Add WidgetModule and import other modules that used in widget components into WidgetModule